src/block/timeline/edit.js

```diff
@@ -112,6 +135,149 @@ const Edit = props => {
 		'stk-block-content',
 	], getContentAlignmentClasses( props.attributes ) )
 
+	// get previous and next elements/blocks
+	const getAdjacentBlocks = () => {
```

Comment:
You can remove this because the `nextBlock` and `previousBlock` are now readily available